packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -916,7 +916,18 @@ export const formatters: Formatters = {
           categoryString = 'SQL';
           break;
       }
-      return `User [${user}] has ran a query [${query}] in category [${categoryString}] against [${target}] on [${db_service}], request path: [${path}]`;
+
+      let message = `User [${user}] has ran a [${categoryString}] query in [${db_service}], request path: [${path}]`;
+
+      if (query != '') {
+        message = message + `, query string: [${truncateStr(query, 80)}]`;
```

Comment:
Nit: 
```typescript
if (query) {
  message += `, query string: [${truncateStr(query, 80)}]`;
}
```
(same below)

Reply:
`if` can be also shortened :)

Reply:
All right, I can see this is the idiomatic way of checking for empty string, done.